package.json

```diff
@@ -17,18 +17,28 @@
   },
   "pnpm": {
     "overrides": {
-      "minimatch@^3.0.4": "^3.1.2",
-      "svelte-hmr": "workspace:*"
+      "svelte-hmr": "workspace:*",
+      "semver@<7.5.2": ">=7.5.2",
+      "svelte": "$svelte"
     },
     "peerDependencyRules": {
-      "ignoreMissing": ["nollup","webpack","webpack-dev-server"]
+      "ignoreMissing": [
+        "nollup",
```

Comment:
is this still needed? I'm curious what's requiring it

Reply:
Indeed, these peer dependencies on the dev server / bundler have long been removed. I'm removing this too.